src/features/transfer/useTokenTransfer.ts

```diff
@@ -108,10 +110,12 @@ async function executeTransfer({
 
     const warpCore = getWarpCore();
 
-    const isCollateralSufficient = await warpCore.isDestinationCollateralSufficient({
-      originTokenAmount,
-      destination,
-    });
+    const isCollateralSufficient = SKIP_DEST_BALANCE_CHECK_STANDARDS.includes(originToken.standard)
+      ? true
+      : await warpCore.isDestinationCollateralSufficient({
+          originTokenAmount,
+          destination,
+        });
```

Comment:
I recommend putting this kind of logic in the WarpCore. Knowning which standards need special exceptions is not a UI-level concern.